support-frontend/assets/pages/[countryGroupId]/components/oneTimeCheckoutComponent.tsx

```diff
@@ -327,7 +328,7 @@ export function OneTimeCheckoutComponent({
 			if (validityState.valueMissing) {
 				setAction(missing); // required
 			} else {
-				setAction(invalid ?? ' '); // pattern mismatch
+				setAction(invalid ?? 'Invalid input'); // pattern mismatch
```

Comment:
@paul-daniel-dempsey felt it would be better to fallback to a very general error message, rather than an empty error message here.